requirements/edx/base.in

```diff
@@ -35,7 +35,7 @@ celery==3.1.25                      # Asynchronous task execution library
 defusedxml==0.4.1                   # XML bomb protection for common XML parsers
 Django==1.11.16                     # Web application framework
 django-babel-underscore             # underscore template extractor for django-babel (internationalization utilities)
-django-config-models                # Configuration models for Django allowing config management with auditing
+django-config-models>=0.2.2         # Configuration models for Django allowing config management with auditing
```

Comment:
are we sure we want >= and not == here?  
I think we don't commonly use >=

Reply:
Version 0.2.2 has the code that I want. I think using `>=` makes that the clearest.

Reply:
why not django-config-models==0.2.2?  
it is clear you want that version and there is no risk of things breaking due to patches

Reply:
Because we want to allow upgrades to happen. We only pin input versions as required to not break. In this case, I want to specify that edx-platform won't work with earlier versions of config models (because I'm about to introduce a config model that uses a 0.2.2 feature), but I expect it to keep working as config-model is upgraded.

Reply:
Abstractly I don't think I agree, but in practice I think this is fine